### PR TITLE
Add support for AdminEnvirons

### PR DESCRIPTION
**Pull request by LtRipley36706**
Add support for AdminEnvirons
  - Wired up support commands `setlbenviron` and `setglobalenviron`
  - `setglobalenviron` is operates exactly as seen in retail worlds. in that it affects all players globally throughout the world, including bug with dungeons.
  - `setlbenviron` operates differently in that it localizes effect to landblock and immediate adjacents for area-based events.

### CODE REVIEW

File: Source/ACE.Entity/Enum/EnvironChangeType.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+namespace ACE.Entity.Enum
+{
+    public enum EnvironChangeType
+    {
+        Clear               = 0x00,
```
Comment (gmriggs) [inline Source/ACE.Entity/Enum/EnvironChangeType.cs]: enum duplicated from existing ACE.Server.Network.Enum.EnvironChangeType

File: Source/ACE.Entity/Enum/EnvironChangeType.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        Thunder2Sound       = 0x77,
+        Thunder3Sound       = 0x78,
+        Thunder4Sound       = 0x79,
+        Thunder5Sound       = 0x7A,
+        Thunder6Sound       = 0x7B
+    }
+
+    public static class EnvironChangeTypeExtensions
+    {
+        public static bool IsFog(this EnvironChangeType type)
```
Comment (gmriggs) [inline Source/ACE.Entity/Enum/EnvironChangeType.cs]: Clear will be flagged as Fog?

File: Source/ACE.Entity/Enum/EnvironChangeType.cs (hunk 1 above)
Comment (LtRipley36706) [reply Source/ACE.Entity/Enum/EnvironChangeType.cs]: deleted the one from network

File: Source/ACE.Entity/Enum/EnvironChangeType.cs (hunk 2 above)
Comment (LtRipley36706) [reply Source/ACE.Entity/Enum/EnvironChangeType.cs]: clear is a "fog" choice

**Pull request merged**